graphics/macgui/macwindowborder.cpp

```diff
@@ -200,10 +200,24 @@ void MacWindowBorder::drawTitle(ManagedSurface *g, MacWindowManager *wm, int tit
 }
 
 void MacWindowBorder::setBorderType(int type) {
+	updateBorderOffsets(type);
 	_useInternalBorder = true;
 	_borderType = type;
 }
 
+void MacWindowBorder::updateBorderOffsets(int type) {
+	_borderOffsets.left = borders[type].offsets.left;
```

Comment:
You could assign borders[type].offsets to a local reference, to simplify the code in this function
e.g. something like:

    BorderOffsets &offsets = borders[type].offsets;